package.json

```diff
@@ -45,7 +45,7 @@
     "@angular/platform-browser": "^17.3.11",
     "@angular/platform-browser-dynamic": "^17.3.11",
     "@angular/router": "^17.3.11",
-    "@fortawesome/fontawesome-pro": "^6.4.0",
+    "@fortawesome/fontawesome-pro": "./vendor-packages/fontawesome-pro-6.5.2.tgz",
```

Comment:
This isn't necessary. You can pull the package directly from npm

Reply:
I was getting an error for some reason so resorted to this. Will try again.

Reply:
Your global npmrc should have the fontawesome registry details (including auth token) in it.

**You should not include these in the repo-level npmrc like Spike as this is a public repo and our token is private**